run/custom_domain_mapping/main.tf

```diff
@@ -17,14 +17,14 @@
 # [START cloudrun_custom_domain_mapping_parent_tag]
 # [START cloudrun_custom_domain_mapping_run_service]
 resource "google_cloud_run_v2_service" "default" {
-  name     = "custom-domain"
+  name     = "your-service-name" # Replace with your desired service
   location = "us-central1"
 
   deletion_protection = false # set to true to prevent destruction of the resource
 
   template {
     containers {
-      image = "us-docker.pkg.dev/cloudrun/container/hello"
+      image = "REGION-docker.pkg.dev/PROJECT_ID/REPOSITORY/IMAGE" # Replace with your container image path
```

Comment:
All samples in this repo are tested by validating the Terraform code, and most samples are additionally tested by applying the Terraform code to create the defined resource. 

In either case, all values being used in resource arguments should be valid. In this case, we always use the "hello" image because that image exists (and is used as the "sample container" when deploying a Cloud Run service). Your change here will never be valid as `REGION` is not a valid region (at minimum). 

As with the above comment, restore the original image value, and adjust the comment to say "container image" rather than "path".